google/resource_container_cluster.go

```diff
@@ -394,6 +436,47 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		cluster.NodePools = nodePools
 	}
 
+	if v, ok := d.GetOk("ip_allocation_policy"); ok {
+		cluster.IpAllocationPolicy, err = expandIPAllocationPolicy(v)
+		if err != nil {
+			return err
+		}
+
+		region := getRegionFromZone(zoneName)
```

Comment:
I'm not sure we actually need these checks inside of Terraform. If I get rid of them I get this error from the API: `Google Compute Engine: services secondary range "pods" not found in subnet "container-net-asdfafw"` which serves the same purpose

Reply:
Sounds good to me.